CHANGELOG.md

```diff
@@ -8,33 +8,55 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
-## [0.1.0] - 201y-mm-dd
+## [0.1.0] - 2013-10-20
 
 ### Added
--
--
--
-
-### Changed
--
--
--
-
-### Deprecated
--
--
--
-
-### Removed
--
--
--
-
-### Fixed
--
--
--
 
+#### Native sniffs
+- `WPThemeReview.CoreFunctionality.FileInclude`: checks if a theme uses `include(_once)` or `require(_once)` when `get_template_part()` should be used. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features).
+- `WPThemeReview.CoreFunctionality.NoDeregisterCoreScript`: checks if a theme deregistered core script (javascript). [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#stylesheets-and-scripts).
+- `WPThemeReview.CoreFunctionality.NoFavicon`: checks if a theme is hard coding favicons instead of using core implementation. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features).
+- `WPThemeReview.CoreFunctionality.NoTitleTag`: checks if a theme is using a `<title>` tag instead of `add_theme_support( 'title-tag' )`. `<svg>` tag can use a `<title>` tag. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features).
+- `WPThemeReview.PluginTerritory.AdminBarRemoval`: checks if a theme is removing admin bar. This rule was pulled from `WPCS` and will be removed in version 2.0 of `WPCS`. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features).
+- `WPThemeReview.PluginTerritory.ForbiddenFunctions`: checks if a theme is using functions that fall under a plugin territory. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#presentation-vs-functionality).
+- `WPThemeReview.PluginTerritory.NoAddAdminPages`: checks if a theme is using `add_..._page()` functions, with the exception of `add_theme_page()`. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/theme-check-plugin/#admin-menu).
+- `WPThemeReview.PluginTerritory.SessionFunctionsUsage`: taken from `WordPress-VIP` ruleset, which will be deprecated in the `WPCS` version 2.0. Prevents the usage of the session functions in themes.
+- `WPThemeReview.PluginTerritory.SessionVariableUsage`: taken from `WordPress-VIP` ruleset, which will be deprecated in the `WPCS` version 2.0. Prevents the usage of the session variables in themes.
+- `WPThemeReview.Plugins.CorrectTGMPAVersion` Verifies that if the [TGM Plugin Activation](http://tgmpluginactivation.com/) library is included, the correct version is used. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/theme-check-plugin/#admin-menu).
```

Comment:
`Verifies` => `: verifies` (missing the colon and the only explanation starting with a capital)